src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -888,4 +889,12 @@ public function handleLevelSoundEvent(LevelSoundEventPacket $packet) : bool{
 		 */
 		return true;
 	}
+
+	public function handleEmote(EmotePacket $packet) : bool {
+		if($this->player->getServer()->getTick() - $this->player->lastEmoteTick > 5){
+			$this->player->getWorld()->broadcastPacketToViewers($this->player->getPosition(), EmotePacket::create($packet->getActorRuntimeId(), $packet->getEmoteId(), $packet::FLAG_SERVER));
```

Comment:
```suggestion
			$this->player->getWorld()->broadcastPacketToViewers($this->player->getPosition(), EmotePacket::create($packet->getActorRuntimeId(), $packet->getEmoteId(), EmotePacket::FLAG_SERVER));
```
Don't use late static binding. It makes usage search harder and prevents optimisations.